glossary.tex

```diff
@@ -201,5 +201,56 @@
 \newglossaryentry{worker-node}
 {
   name={worker node},
-  description={See \gls{compute-node}}
+  description={See \gls{compute-node}},
+}
+\newglossaryentry{OpenStack}
+{
+  name={OpenStack},
+  description={OpenStack is a free and open-source software platform for cloud computing, mostly deployed as infrastructure-as-a-service (IaaS), whereby virtual servers and other resources are made available to customers},
+}
+\newglossaryentry{OpenStack Identity}
+{
+  name={OpenStack Identity},
+  description={OpenStack Identity (Keystone) provides a central directory of users mapped to the OpenStack services they can access. It acts as a common authentication system across the cloud operating system and can integrate with existing backend directory services. It supports multiple forms of authentication including standard username and password credentials, token-based systems and AWS-style logins.},
```

Comment:
explain `AWS` (or remove it)

Reply:
removed AWS, mentioned integration with VSC account.  We could also remove the entry?